crates/runtime/src/read.rs

```diff
@@ -41,3 +46,85 @@ pub fn query(
 
     Ok(blks)
 }
+
+const DEFAULT_REMOTE_PORT: u16 = 9528;
+const DEFAULT_REMOTE_COMPRESSION: &str = "lz4";
```

Comment:
we may use no compression as default? because the ClickHouse seemly change the default compression method for client.

Reply:
Sure, we can.